doc/compiling/windows.md

```diff
@@ -2,67 +2,88 @@
+-   `gmp` and `jsoncpp` are optional, otherwise the bundled versions will be compiled
+-   `gettext` is optional, but required to use translations.
 
+There are other optional libraries, but we don't test if they can build and link correctly.
 
-## Compile Luanti
+Use `--triplet` to specify the target triplet, e.g. `x64-windows` or `x86-windows`.
+-->
 
-### a) Using the vcpkg toolchain and CMake GUI
+### Setup CMake
 
-1. Start up the CMake GUI
-2. Select **Browse Source...** and select DIR/minetest
-3. Select **Browse Build...** and select DIR/minetest-build
+1. Start up the CMake GUI (Win > search "cmake-gui" > open)
+2. Select **Browse Source...** and select `path/to/minetest` (where you've cloned the repo)
+3. Select **Browse Build...** and select `path/to/minetest/build` (a new folder that CMake will prompt to create)
 4. Select **Configure**
-5. Choose the right visual Studio version and target platform. It has to match the version of the installed dependencies
+5. Choose the right Visual Studio version and target platform. Currently, Luanti uses Visual Studio 16 2019, but newer VS versions should work as well. The VS version has to match the version of the installed dependencies.
 6. Choose **Specify toolchain file for cross-compiling**
 7. Click **Next**
-8. Select the vcpkg toolchain file e.g. `D:/vcpkg/scripts/buildsystems/vcpkg.cmake`
+8. Select the vcpkg toolchain file e.g. `C:/vcpkg/scripts/buildsystems/vcpkg.cmake`
 9. Click Finish
-10. Wait until cmake have generated the cash file
+10. Wait until CMake generates the cache file (this may take about 10-30 minutes, depending on your device)
 11. If there are any errors, solve them and hit **Configure**
 12. Click **Generate**
 13. Click **Open Project**
-14. Compile Luanti inside Visual studio.
 
-### b) Using the vcpkg toolchain and the commandline
+## Compile Luanti
+
+There are two ways to compile Luanti: via Visual Studio or via CLI.
+
+### Compile in Visual Studio
 
-Run the following script in PowerShell:
+14. Compile Luanti inside Visual Studio.
+    - If you get "Unable to start program '...\x64\Debug\ALL_BUILD'. Access is denied", try compiling via the CLI instead.
+
+### Compile via CLI
+
+While in the `path/to/minetest` folder, run the following script in PowerShell:
 
 ```powershell
-cmake . -G"Visual Studio 16 2019" -DCMAKE_TOOLCHAIN_FILE=D:/vcpkg/scripts/buildsystems/vcpkg.cmake -DCMAKE_BUILD_TYPE=Release -DENABLE_CURSES=OFF
+vs="Visual Studio 16 2019" # or "Visual Studio 17 2022", whatever matches your system
+toolchain_file="C:/vcpkg/scripts/buildsystems/vcpkg.cmake" # ensure this path is correct
+cmake . -G $vs -DCMAKE_TOOLCHAIN_FILE=$toolchain_file -DCMAKE_BUILD_TYPE=Release -DENABLE_CURSES=OFF
 cmake --build . --config Release
 ```
-Make sure that the right compiler is selected and the path to the vcpkg toolchain is correct.
-
 
 ## Windows Installer using WiX Toolset
```

Comment:
Any advice on what to do with this "Windows Installer" section? I'm not sure what role it plays, or when it should come in. It mentions "compil[ing] binaries as above", but I don't remember seeing any "RUN_IN_PLACE" option

Reply:
The `RUN_IN_PLACE` cache variable is used by Luanti to determine what paths to use for resources such as worlds and mods. I think I used the installer before, but it's been a while. We might have to rediscover how this works if no one remembers.

Reply:
Resolving for now as out-of-scope for this PR